src/styles/styles.js

```diff
@@ -2874,6 +2874,37 @@ const styles = {
         paddingTop: 80,
         paddingBottom: 45,
     },
+
+    joinRoomPromptContainer: {
+        maxHeight: 92,
```

Comment:
I don't think we need to set a max height here. What is this doing exactly?

Reply:
Ah this was leftover from some debugging I was doing, it should be removed now!